slider/beatmap.py

```diff
@@ -1871,6 +2066,38 @@ def ar(self,
 
         return ar
 
+    @lazyval
+    def breaks(self):
+        """The breaks with all other events filtered out.
+        """
+        return tuple(e for e in self.events if isinstance(e, Break))
+
+    @lazyval
+    def background(self):
+        """The background, if it exists, otherwise returns None.
+        """
+        return next(
+            (e for e in self.events if isinstance(e, Background)), None)
+
+    @lazyval
+    def videos(self):
+        """The videos with all other events filtered out.
+        """
+        return tuple(e for e in self.events if isinstance(e, Break))
+
+    @lazyval
+    def hit_objects_no_spinners(self):
```

Comment:
where is this method used? I see that `circles` was used in performance points calculation, so it's good that you added that back, but I can't see where `hit_objects_no_spinners` is used.

Reply:
I dont think it's being used. I just copied it back i guess cause i thought it was going to get used.